src/views/Home.vue

```diff
@@ -198,6 +198,17 @@
     color: #34495e;
     text-decoration: none;
   }
+  .page {
+    position: absolute;
```

Comment:
@egoist you can also use `margin-left: 280px` instead of `position: absolute` here, I chose `position` just because I like it XD